docs/builders.rst

```diff
@@ -163,3 +163,21 @@ or
 .. hint::
 
     As an alternative, you can set the config option :ref:`needs_build_needumls` to export the needumls files during each build.
+
+
+.. _needs_id_builder:
+
+needs_id
+--------
+.. versionadded:: 1.3.0
+
+The **needs_id** builder exports all found needs and selected filter results to a set json files of each need with the name is ``id`` of need.
```

Comment:
I guess the sentence formulation could be improved a bit. It is not very evident what functionality is being provided here.

I understand that the builder will create a folder and create single json files for each need. Is this correct?

Reply:
This is correct. In the docs sphinx-needs we don't use it so needs_build_json_per_id  always set = True. In the Pannel function we also set needs_build_json_per_id  = True to solve about read I/O from big json file to seperate minor json file

Reply:
Such things I will reformulate after the PR got merged ;)